runtime/description.js

```diff
@@ -30,7 +30,7 @@ export default class Description {
       return desc;
     }
 
-    return this._recipe.name;
+    return this._recipe.name || '🤷';
```

Comment:
as much as i enjoy icons (and i do!) maybe a more helpful description would be joined particles name or smth? up to you though.